cognite/neat/graph/transformers/_classic_cdf.py

```diff
@@ -3,15 +3,15 @@
 from rdflib import Graph, Literal, URIRef
 
 from cognite.neat.constants import DEFAULT_NAMESPACE
-from cognite.neat.graph.extractors import AssetsExtractor
+from cognite.neat.graph import extractors
 
 from ._base import BaseTransformer
 
 
 class AddAssetDepth(BaseTransformer):
     description: str = "Adds depth of asset in the asset hierarchy to the graph"
     _use_only_once: bool = True
-    _need_changes = frozenset({str(AssetsExtractor.__name__)})
+    _need_changes = frozenset({str(extractors.AssetsExtractor.__name__)})
```

Comment:
Any reason you did not go for this?
```suggestion
    _need_changes = frozenset({extractors.AssetsExtractor})
```